compiler/src/iree/compiler/API/Internal/IREEGPUDialectCAPI.cpp

```diff
@@ -411,6 +413,15 @@ ireeHALExecutableTargetAttrGetGPUTargetInfo(MlirAttribute attr) {
       wgpAttr.getMaxThreadCountPerWorkgroup();
   targetInfo.maxWorkgroupMemoryBytes = wgpAttr.getMaxWorkgroupMemoryBytes();
 
+  if (wgpAttr) {
+    targetInfo.simdsPerWgp = wgpAttr.getSimdsPerWgp();
+  }
+
+  mlir::iree_compiler::IREE::GPU::TargetChipAttr chipAttr = gpuTargetAttr.getChip();
+  if (chipAttr) {
```

Comment:
If this variable is not used outside the `if`, you can declare it in the condition
```suggestion
  if (mlir::iree_compiler::IREE::GPU::TargetChipAttr chipAttr = gpuTargetAttr.getChip()) {
```

Reply:
[a2070b1](https://github.com/iree-org/iree/pull/22527/commits/a2070b1a55ce5ecc07fe2729560d7ff6a76a81a0)